CHANGELOG.md

```
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+## v4.51.1
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)